linux/power_operation_scripts/set_power_script_default.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Set and check default power operation script
+# # Parameters:
+#   power_script_op: power operation from command vmware-toolbox-cmd
+# # Return:
+#   script_file_path: path of default script file
+#   script_file_in_conf_file: script file in tools.conf
+
+- name: "Get default power script file path"
+  ansible.builtin.command: "{{ vmware_toolbox_cmd_path }} script {{ power_script_op }} default"
+  register: power_script_path_result
+  delegate_to: "{{ vm_guest_ip }}"
+
+- name: "Set fact of default power script file path"
+  ansible.builtin.set_fact:
+    script_file_path: "{{ power_script_path_result.stdout.strip() }}"
+    script_file_in_conf_file: "{{ power_script_path_result.stdout.strip() | basename }}"
+  when: power_script_path_result.stdout is defined
```

Comment:
`script_file_path` and `script_file_in_conf_file` must be initialized at the beginning of this task file